packages/@lwc/engine-core/src/framework/template.ts

```diff
@@ -137,13 +200,13 @@ if (process.env.NODE_ENV === 'test-karma-lwc') {
 
 function buildParseFragmentFn(
     createFragmentFn: (html: string, renderer: RendererAPI) => Element
-): (strings: string[], ...keys: number[]) => () => Element {
-    return (strings: string[], ...keys: number[]) => {
+): (strings: string[], ...keys: (string | number)[]) => () => Element {
+    return (strings: string[], ...keys: (number | string)[]) => {
```

Comment:
```suggestion
    return (strings: string[], ...keys: (string | number)[]) => {
```

We use `string | number` above; let's be consistent.

Reply:
[Updated](https://github.com/salesforce/lwc/pull/4056/commits/93f51d8c0d62013ab61fb6f499ce79329df14da1#diff-9d36a520650dfa1dd13874ab3870b6f688303b9e5860066178098aadcf45a860L204)!